### PR TITLE
Add some games

### PR DESCRIPTION
**Pull request by M0REKZ**
This adds 3 different things (2 of these also adds an original Game)

### CODE REVIEW

File: originals/t.yaml [hunk 1]
```diff
@@ -54,6 +54,23 @@
   - OSX
   - Windows
 
+- name: Teeworlds
```
Comment (cxong) [inline originals/t.yaml:57]: OSGC doesn't list original open source games, so teeworlds should not be added

Comment (cxong) [reply originals/t.yaml:57]: for original open source games consider contributing to projects like https://trilarion.github.io/opensourcegames

Comment (M0REKZ) [reply originals/t.yaml:57]: What about DDraceNetwork? I was considering to add it here too, it's a teeworlds fork that is being distributed as a standalone Game on Steam

Comment (cxong) [reply originals/t.yaml:57]: it shouldn't be added either imo

Comment (M0REKZ) [reply originals/t.yaml]: Ok

**Pull request merged**